manim/mobject/graphing/coordinate_systems.py

```diff
@@ -1836,12 +1836,19 @@ def construct(self):
 
         return T_label_group
 
-    def __matmul__(self, coord: Point3D | Mobject):
+    def __matmul__(self, coord: Iterable[float] | Mobject):
```

Comment:
I left a suggestion regarding the return type.

About the parameter type, I'm not sure. The Liskov substitution principle says that, when overriding methods, parameter types are contravariant and return types are covariant, which means that the parameter type cannot get more specific when overriding a method. It can only get broader or stay the same. The reverse is true for return types.

Maybe the parameter type could be `Mobject`, because all of the coordinate systems accept a Mobject as an argument (except for `ComplexPlane`, but that should be changed in another PR).

```suggestion
    def __matmul__(self, coord: Iterable[float] | Mobject) -> Point3D:
```